statsmodels/tsa/arima_model.py

```diff
@@ -925,7 +927,10 @@ def fit(self, start_params=None, trend='c', method="css-mle",
             self.nobs = len(self.endog) - k_ar
 
         if start_params is not None:
-            start_params = np.asarray(start_params)
+            if hasattr(start_params, '__call__'):
```

Comment:
The [sourceforge site](http://statsmodels.sourceforge.net/devel/install.html) lists dependencies as
> Python >= 2.6, including Python 3.x

which made me assume statsmodels would work with Python 3.0-3.2. I'm curious, is it documented somewhere besides this thread that these versions aren't fully supported?